### PR TITLE
Update:Make streaming responses more like the OpenAI API.

### PR DESCRIPTION
**Pull request by kncxstudio**
Update:Make streaming responses more like the OpenAI API.

### CODE REVIEW

**Comment by nacho00112** [comment]
Why you didn't used the `string` standard library to generate the `completion_id`?

**Comment by iloveyumyumshrimpnoodles** [comment]
> Why you didn't used the `string` standard library to generate the `completion_id`?

Why import another entire library, just for a few characters?

File: interference/app.py [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
-    response = ChatCompletion.create(model=model, stream=streaming,
+
+    response = ChatCompletion.create(provider=Provider.You, model=model, stream=streaming,
                                      messages=messages)
-    
+
     if not streaming:
         while 'curl_cffi.requests.errors.RequestsError' in response:
-            response = ChatCompletion.create(model=model, stream=streaming,
+            response = ChatCompletion.create(provider=Provider.You, model=model, stream=streaming,
```
Comment (w3bdesign) [inline interference/app.py:27]: Why hardcode model here?

**Review by w3bdesign — CHANGES_REQUESTED** [review]
Why hardcode model?

**Pull request closed without merging**